.github/workflows/generated-windows-binary-conda-nightly.yml

```diff
@@ -256,1817 +256,7 @@ jobs:
       aws-pytorch-uploader-secret-access-key: ${{ secrets.AWS_PYTORCH_UPLOADER_SECRET_ACCESS_KEY }}
       conda-pytorchbot-token: ${{ secrets.CONDA_PYTORCHBOT_TOKEN }}
     uses: ./.github/workflows/_binary-upload.yml
-  conda-py3_7-cuda11_6-build:
```

Comment:
why are we removing cuda11_6? Is it because it is already default?

Reply:
I think this is due to the error above with ```arches += CUDA_ARCHES``` for windows. Should be fixed now